content/translation.js

```diff
@@ -0,0 +1,126 @@
+const LRU = require("lru-cache");
+
+const Parser = require("../kumascript/src/parser.js");
+
+const cacheKSMacros = new LRU({ max: 2000 });
+
+const IMPORTANT_MACROS = new Map(
+  [
+    "Compat",
+    "EmbedInteractiveExample",
+    "EmbedLiveSample",
+    "EmbedGHLiveSample",
+    "CSSSyntax",
+    "SeeCompatTable",
+    "languages",
+    "AvailableInWorkers",
+    "IncludeSubnav",
+    // XXX List all the important sidebar macros??
```

Comment:
@peterbe I think all of the important sidebar macros that we should include in this list are as follows:
- AddonSidebarMain
- CanvasSidebar
- CSSRef
- DefaultAPISidebar
- GamesSidebar
- HTMLSidebar
- HTTPSidebar
- JsSidebar
- LearnSidebar
- MDNSidebar
- ToolsSidebar
- WebAssemblySidebar
- WebExtAPISidebar
- WebGLSidebar
- WebRTCSidebar

Did I miss any @wbamberg?

Reply:
@escattone APIRef is one that springs to mind.

Reply:
I'm not sure what makes a macro "important" in this context. The old [macro massacre sheet](https://docs.google.com/spreadsheets/d/1Q34ivQMlVqbDiM8-jiXZLE8ycbBNe-WDUZboD4Xk42M/edit?ts=60539058#gid=0) listed these as sidebars:

```

AddonSidebar
--
APIRef
CanvasSidebar
CSSRef
CSSTutorialTOC
DefaultAPISidebar
DocStatusQuickLinks
EditorGuideQuicklinks
FirefoxSidebar
GamesSidebar
HTMLRef
HTMLSidebar
HTTPSidebar
IncludeSubnav
JSRef
JsSidebar
LearnSidebar
ListSubpagesForSidebar
MathMLRef
MDNSidebar
QuickLinksWithSubpages
ServiceWorkerSidebar
SVGRef
ToolsSidebar
WebAssemblySidebar
WebExtAPISidebar
WebGLSidebar
WebRTCSidebar
```

Apart from `APIRef`, the most notable ones missing from your list are `HTMLRef` (the main HTML sidebar) and `JSRef` (the other JS sidebar).

Reply:
Thanks @wbamberg and @chrisdavidmills!